File: Development/WoodLotBackEnd/src/main/java/it/unisa/WoodLot/model/entity/ProdottoCarrello.java
package it.unisa.WoodLot.model.entity;

import lombok.Data;

import javax.persistence.*;
import java.io.Serializable;
import java.util.List;

/**
 * Classe che modella un prodotto di un carrello
 *
 * @author Alessia Ture
 */
@Data
@Entity
@Table(name = "prodotto_carrello")
public class ProdottoCarrello implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @ManyToOne()
    private Albero albero;

}

File: Development/WoodLotFrontEnd/src/app/entita/prodottoCarrello/prodotto-carrello.ts
import {Albero} from "../albero/albero";
import {Carrello} from "../carrello/carrello";

export class ProdottoCarrello {
  public albero: Albero
  public id: number


  constructor(albero: Albero, carrello: Carrello) {
    this.id = 0
    this.albero = albero

  }
}
